cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -605,3 +615,24 @@ message VersionReference {
   // for a flux v2 and Carvel it’s a semver expression, such as ">=10.3 < 10.4"
   string version = 1;
 }
+
+// InstalledPackageStatus
+//
+// An InstalledPackageStatus reports on the current status of the installation.
+message InstalledPackageStatus {
+  // Ready
+  //
+  // An indication of whether the installation is ready or not
+  bool ready = 1;
+
+  // Reason
+  //
+  // A single word indicating the reason for the current status. For example, "Pending",
+  // "InstallSucceeded", or "InstallFailed"
+  string reason = 2;
```

Comment:
I guess this will be (in the helm-direct) something like this: https://github.com/helm/helm/blob/main/pkg/release/status.go#L23

Just thinking off the top of my mind: would including a list of every possible state somewhere be useful? I don't really think so. Just perhaps adding it as an `enum` in the code implementation, but not a user-facing thing.

Reply:
A second thought: perhaps a kind of standardization on the statuses is, indeed, required, no? If not, how can we know if a package is OK/pending/failed/etc from the core API ?